### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ro4i7**
## Description

[Update CONTRIBUTING.md .]

## Changes Made
[i have updated the CONTRIBUTING.md as necessary ]

## Checklist
- [x] I have read and followed the contributing guidelines.
- [ ] I have tested my changes thoroughly and they work as expected.
- [ ] I have added necessary tests for the changes made.
- [ ] I have updated the documentation to reflect the changes made.
- [x] My code follows the project's coding style and standards.
- [x] I have added appropriate commit messages and comments for my changes.

## Related Issues
[If your changes relate to any existing issues, please list them here.]

## Additional Notes
[Provide any additional notes or context that may be helpful for the reviewers.]

Feel free to modify and customize this template as needed to fit the specific needs of the GCP Scanner project.

### CODE REVIEW

**Comment by ro4i7** [comment]
Hello, @ZetaTwo I have updated. if there is any issue, please tag me. Thank you.

**Comment by mshudrak** [comment]
Please resolve the conflict. We had quite similar PR recently merged.

**Comment by ro4i7** [comment]
> Please resolve the conflict. We had quite similar PR recently merged.

yeah, resolved the conflict. I have done this PR yesterday, but I committed to two solutions in a single PR.

**Comment by ro4i7** [comment]
> > Please resolve the conflict. We had quite similar PR recently merged.
> 
> yeah, resolved the conflict. I have done this PR yesterday, but I committed to two solutions in a single PR.

if possible, please merge it to the `main`.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,3 +81,4 @@ We believe in creating a friendly and welcoming environment for all members of o
 
 Please read the code of conduct carefully and make sure you understand what is expected of you. If you have any questions or concerns, please contact the project maintainers by posting to the [Google Group](https://groups.google.com/g/gcp-scanner).
 
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:84]: Unnecessary new line

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ We'd love to accept your patches and contributions to this project. There are ju
 
 ## Contributor License Agreement
 
-Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to [CLA](https://cla.developers.google.com/) to see your current agreements on file or to sign a new one.
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:7]: Unnecessary new line

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ We'd love to accept your patches and contributions to this project. There are ju
...
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:9]: Unnecessary new line

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -56,6 +58,13 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
 - Use descriptive names that reflect the purpose of the branch.
 - Add `issue tracker id`
 
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:61]: Unnecessary new line

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -56,6 +58,13 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
...
+**Example:** `feat/<issue-id>/gcp-compute-engine-support`
```
Comment (mshudrak) [inline CONTRIBUTING.md:62]: You need to have new line in order to have it as a list and not like a sentence. Additionally, you have duplicate feat/issue-id now. PTAL at the final result.
https://github.com/ro4i7/gcp_scanner/blob/patch1/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -56,6 +58,13 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
...
+**Example:** `docs/11/update-installation-instructions`
+**Example:** `fix/87/scan-logic-bug-fix`
+**Example:** `test/62/new-scan-feature-tests`
+**Example:** `ci/33/new-dependency-build-pipeline`
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:67]: Unnecessary new line

**Review by mshudrak — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -70,5 +72,6 @@ If you need help or have any questions about the project, please feel free to:
 
 We believe in creating a friendly and welcoming environment for all members of our community. As such, we follow the [Google Open Source Community Guidelines](https://opensource.google/conduct/), which outlines our expectations for all those who participate in our project.
 
-Please read the code of conduct carefully and make sure you understand what is expected of you. If you have any questions or concerns, please contact the project maintainers by posting to the [Google Group](https://groups.google.com/g/gcp-scanner).
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:75]: Why removing that?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -70,5 +72,6 @@ If you need help or have any questions about the project, please feel free to:
...
+
```
Comment (mshudrak) [inline CONTRIBUTING.md:76]: Unnecessary new line

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -53,10 +53,12 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
 
 - Use lowercase letters.
 - Separate words with hyphens.
-- Use descriptive names that reflect the purpose of the branch.
```
Comment (mshudrak) [inline CONTRIBUTING.md:56]: Why removing that?

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -4,7 +4,7 @@ We'd love to accept your patches and contributions to this project. There are ju
 
 ## Contributor License Agreement
 
-Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to [CLA](https://cla.developers.google.com/) to see your current agreements on file or to sign a new one.
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution; this simply gives us permission to use and redistribute your contributions as part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
```
Comment (mshudrak) [inline CONTRIBUTING.md:7]: I don't think we need to change the link

**Review by mshudrak — CHANGES_REQUESTED** [review]
I am not sure I am following what you want to change with that PR. Are you sure we need that change at all?

**Comment by ro4i7** [comment]
> I am not sure I am following what you want to change with that PR. Are you sure we need that change at all?

@mshudrak  Changed, please check.

**Review by mshudrak — CHANGES_REQUESTED** [review]
Still has some minor issue with text. right now it is

Example: feat//gcp-compute-engine-support.

Please take a look at your final result before submitting PR.

**Review by mshudrak — APPROVED** [review]

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -54,9 +54,11 @@ This project follows [Google's Open Source Community Guidelines](https://opensou
 - Use lowercase letters.
 - Separate words with hyphens.
 - Use descriptive names that reflect the purpose of the branch.
-- Add `issue tracker id`
```
Comment (mshudrak) [inline CONTRIBUTING.md:57]: We still want this line.

**Review by mshudrak — CHANGES_REQUESTED** [review]
Sorry, approved by mistake. PTAL at my comment.

**Review by mshudrak — APPROVED** [review]

**Review by mshudrak — APPROVED** [review]

**Pull request merged**